### PR TITLE
docs(admob): update docs to reference new external plugin

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

Points to [remote source](https://github.com/googleads/googleads-mobile-flutter) following deprecation of admob package from this repository.
## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: website/plugins.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ module.exports = [
     name: 'AdMob',
     pub: 'firebase_admob',
```
Comment (Salakar) [inline website/plugins.js:4]:
```suggestion
    pub: 'google_mobile_ads',
```

File: website/plugins.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ module.exports = [
...
     firebase: 'admob',
+    remoteSource:'https://github.com/googleads/googleads-mobile-flutter',
```
Comment (Salakar) [inline website/plugins.js:6]:
```suggestion
    remoteSource: 'https://github.com/googleads/googleads-mobile-flutter',
```

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ interface Plugin {
   pub: string;
   documentation: string;
   firebase: string;
+  remoteSource?:string;
```
Comment (Salakar) [inline website/src/pages/index.tsx:19]:
```suggestion
  remoteSource?: string;
```

**Comment by Salakar** [comment]
https://github.com/FirebaseExtended/flutterfire/blob/5fc14240f42c1cfc578a01510b917c7eddf4b6a9/docs/versions.js#L5-L6

These will need removing also I think

**Comment by Salakar** [comment]
Another one sorry:

https://github.com/FirebaseExtended/flutterfire#firebase_admob

Lets remove this one entirely from the readme

**Comment by rrousselGit** [comment]
Screenshot 2021-03-19 at 15 39 23


Is there a way to add the pub badge and link to the doc?

File: website/src/pages/index.tsx [hunk 4]
```diff
@@ -99,10 +100,10 @@ function Home() {
                     <strong>{plugin.name}</strong>
                   </td>
                   <td style={{ minWidth: 150 }}>
-                    <img
+                    {!plugin.remoteSource && <img
```
Comment (Salakar) [inline website/src/pages/index.tsx:103]: What's the reason for removing the pub badge on this one? Should still be working, e.g.: https://img.shields.io/pub/v/google_mobile_ads.svg

sfsf

Comment (russellwheatley) [reply website/src/pages/index.tsx:103]: huh, I thought it wasn't working. will update.

Comment (Salakar) [reply website/src/pages/index.tsx:103]: Thanks!

File: website/plugins.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 module.exports = [
   {
     name: 'AdMob',
-    pub: 'firebase_admob',
+    pub: 'google_mobile_ads',
     firebase: 'admob',
+    remoteSource: 'https://github.com/googleads/googleads-mobile-flutter',
     documentation:'',
```
Comment (Salakar) [inline website/plugins.js:7]:
```suggestion
    documentation: 'https://pub.dev/documentation/google_mobile_ads/latest/',
```

As suggested by @rrousselGit

**Review by rrousselGit — APPROVED** [review]

**Review by Salakar — APPROVED** [review]
LGTM, :shipit:

**Pull request merged**